ramandecompy/peakidentify.py

```diff
@@ -612,6 +610,7 @@ def process_score(unknown_peaks,known_peaks,k, precision, unknownname, knownname
                             columns=[str(unknownname)+'_vs_'+str(knownname)+'_peak_Scores Unnormalized'])
         compdf=compdf.assign(Peaks=score_sort(unknown_peaks,known_peaks,1, precision)[0][1][:])
     return compdf
+
 def score_table(unknown_peaks,known_peaks, precision,unknownname,knownname):
     "documentation"
```

Comment:
need doc string here, please keep to format laid out -- see parker's functions for good doc string examples. should include defining the input, and return and purpose

Reply:
made doc string